### PR TITLE
Removes "recording" from verbage

### LINKED ISSUE #163: Remove "record" from verbage
A while ago we made the decision to remove "record" from our in-app copy to distance our application language from the assumptions people may carry over due to the term's use in time-based media.

### Requirements
- Replace "record" with more suitable language in both the application & documentation.
- Update copy to align any finalized terms with Browsertrix Cloud
  - "Collection" is specifically used kinda differently over here, may need some discusion)

### PR DESCRIPTION
**Pull request by Shrinks99**
Closes #163

- Replaces "recording" with "archiving" when referring to the action of collecting content from the web
- Replaces "archive" with "archived item" and "archiving session" when referring to the object
- Removes ellipses on button rollover text
- Other minor copy changes

### Testing

Visit every page in the app, read the copy, and ensure it makes sense and is consistent?

### CODE REVIEW

File: src/ui/coll-info.js [hunk 1]
```diff
@@ -311,14 +311,14 @@ class WrRecCollInfo extends CollInfo
     <wr-modal bgClass="has-background-warning" @modal-closed="${() => this.shareWarn = false}" title="Start Sharing?">
       <div class="content is-size-7">
         <p>
-          Do you want to share the all the pages in the archive "<i>${this.coll.title}</i>" via IPFS, a peer-to-peer
+          Do you want to share the all the content in "<i>${this.coll.title}</i>" via IPFS, a peer-to-peer
           distributed storage network?
         </p>
-        <p>Your archive will have a unique link which can be shared with others to load your archive
+        <p>Your archiving session will have a unique link which can be shared with others to load it 
         via ReplayWeb.page. All archived data is loaded on-demand when replayed. This feature is experimental and likely works best with smaller archives.</p>
```
Comment (tw4l) [inline src/ui/coll-info.js:318]:
```suggestion
        <p>Your archiving session will have a unique link which can be shared with others to load and replay
        on-demand in ReplayWeb.page. This feature is experimental and likely works best with smaller archives.</p>
```

File: src/ui/coll-info.js [hunk 2]
```diff
@@ -311,14 +311,14 @@ class WrRecCollInfo extends CollInfo
...
         <p>You can cancel sharing at anytime.
```
Comment (tw4l) [inline src/ui/coll-info.js:319]:
```suggestion
        <p>You can cancel sharing at any time.
```

File: src/ui/app.js [hunk 3]
```diff
@@ -317,26 +317,26 @@ class ArchiveWebApp extends ReplayWebApp
...
+              <button class="button is-small no-pad-mobile" title="Start Archiving" ?disabled="${!this.colls}" @click="${this.onShowStart}">
                 <span class="icon">
                   <fa-icon size="1.0em" aria-hidden="true" .svg="${wrRec}"></fa-icon>
                 </span>
-                <span class="is-hidden-mobile">Start Recording...</span>
+                <span class="is-hidden-mobile">Start Archiving</span>
               </button>
               <div class="rightbar">
-                <div class="infomsg is-hidden-mobile">The ArchiveWeb.page ${IS_APP ? "App" : "Extension"} allows you to create web archives directly in your browser!</div>
+                <div class="infomsg is-hidden-mobile">The ArchiveWeb.page ${IS_APP ? "App" : "Extension"} allows you to archive webpages directly in your browser!</div>
```
Comment (tw4l) [inline src/ui/app.js:339]: Do we have a style guide take on web pages vs. webpages yet? I see them used interchangeably elsewhere and I think either's fine but we might want to make sure we're consistent on that between apps/docs.

File: src/ui/app.js [hunk 4]
```diff
@@ -466,7 +466,7 @@ class ArchiveWebApp extends ReplayWebApp
 
   renderImportModal() {
     return html`
-    <wr-modal style="--modal-width: 740px" @modal-closed="${() => this.showImport = false}" title="Import an Existing Archive">
+    <wr-modal style="--modal-width: 740px" @modal-closed="${() => this.showImport = false}" title="Import Files">
```
Comment (tw4l) [inline src/ui/app.js:469]:
```suggestion
    <wr-modal style="--modal-width: 740px" @modal-closed="${() => this.showImport = false}" title="Import File">
```
I think we only allow one at a time and elsewhere we seem to use singular File

**Review by tw4l — COMMENTED** [review]
Left some suggestions and questions, overall looks nice! Tested from a local build.

File: src/ui/app.js (hunk 3 above)
Comment (Shrinks99) [reply src/ui/app.js:339]: We don't really have a _style guide_ yet!!  There's the documentation one but I've punted doing this to when we eventually have a single component library and therefore a storybook site with the documentation style guide acting as bonus info for documentation only.

I'm a "webpage" guy I think, but super willing to go with whatever we decide on this one... Might continue to punt this to later and make note of it in my Design System Plans obsidian note?

**Review by tw4l — APPROVED** [review]

File: src/ui/app.js [hunk 5]
```diff
@@ -442,22 +442,22 @@ class ArchiveWebApp extends ReplayWebApp
         </div>
         ${IS_APP ? html`
         <label class="checkbox">
-          <input id="preview" type="checkbox"><span>&nbsp;Start in Preview Mode (without recording.)</span>
+          <input id="preview" type="checkbox"><span>&nbsp;Start in Preview Mode (without saving.)</span>
```
Comment (Shrinks99) [inline src/ui/app.js:445]: Good catch!
```suggestion
          <input id="preview" type="checkbox"><span>&nbsp;Start in Preview Mode (without archiving.)</span>
```

**Comment by Shrinks99** [comment]
With Tessa's suggestions and approval I think we can finally put this one to bed!  Any future issues should be able to be dealt with on a case-by-case basis. :)

**Pull request merged**